x-pack/plugin/sql/qa/src/main/resources/docs/docs.csv-spec

```diff
@@ -2095,6 +2097,51 @@ elastic
 ;
 
 
```

Comment:
I would like to see, based on a previous issue fixed in the past, a query where you use a `CONVERT` without the optional parameter for IF, with an alias and grouped by alias: `SELECT CONVERT(IF(languages > 1, IF(languages = 3, '3')), SQL_BIGINT) AS cond FROM test_emp GROUP BY cond ORDER BY cond DESC`.